### DISCUSSION TITLE
Why we use prev_addin in contextual block transformer encoder

### DISCUSSION THREAD

**Discussion by zldzmfoq12** [espnet/espnet#4741] [Q&A]
In the forward_infer function of espnet/espnet2/asr/encoder/contxtual_block_transformer_encoder.py, why do we use prev_addin (previously initialized context embedding vector)? 
Looking at the paper, the initialized context embedding vectors of the previous block are not utilized in the computation of the current block.
Therefore, I wonder what role prev_addin plays in the code.

**Comment by sw005320** [comment]
@eml914, can you answer it for me?
